content/install_maintain/installation/installing-cli.md

```diff
@@ -8,9 +8,9 @@ aliases: /installation/uninstall-cloudify-cli/
 ---
 ## Installing the Cloudify CLI
 
-To remotely connect to Cloudify Manager, you can install the Cloudify Command Line Interface (CLI) on a separate host. The Cloudify CLI includes all of the commands that you require to do any actions on Cloudify Manager.
+To remotely connect to Cloudify Manager, you can install the Cloudify Command Line Interface (CLI) on a separate host. The Cloudify CLI includes all of the commands that required to perform actions on Cloudify Manager.
```

Comment:
\- all of the commands that you require to do any actions
\+ all of the commands necessary to run any actions